tools/mo/openvino/tools/mo/convert_impl.py

```diff
@@ -917,7 +921,10 @@ def _convert(**args):
         init_logger('ERROR', False)
 
         argv.feManager = FrontEndManager()
-        ov_model, legacy_path = driver(argv, {"conversion_parameters": non_default_params})
```

Comment:
please add comments here.
Why can't we check values of parameters during their serialization?

Reply:
I added check here because we can go either legacy path or frontend path. If it is legacy then _tmp IR serializing stucks with model in rt_info, so we need one check there. Another check would be in serialize() method in c++ part. So it looks better to have one check in convert_model.

Reply:
Yes, this check can be done during serialization. I moved it to rt_info serializing.